src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const oneDayRent = 40;
+  let sumForRent = 0;
+
+  if (days >= 7) {
+    sumForRent = oneDayRent * days - 50;
```

Comment:
Avoid using literals in expressions. Define constants for the number of days and the discount amounts to make the code more readable and maintainable, as per the checklist item 2.